scoreboard/logger.py

```diff
@@ -26,7 +26,7 @@ class Formatter(logging.Formatter):
 
     def format(self, record):
         if flask.request:
-            user = ('UID<%d>' % flask.g.uid) if flask.g.uid else '-'
+            user = ('UID<%d>' % flask.g.uid) if 'uid' in flask.g else '-'
```

Comment:
Not sure if you've seen this, but git add -p makes it easy to add partial changes to commits, leaving the reminder to be committed/removed later on. More information at https://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging